### PR TITLE
add role roles

### PR DESCRIPTION
**Pull request by Thiblef**

### CODE REVIEW

**Comment by itkg-alavieille** [comment]
:+1:

**Comment by ggrasset** [comment]
:+1:

File: Backoffice/NavigationPanel/Strategies/AdministrationPanelStrategy.php [hunk 1]
```diff
@@ -42,6 +42,9 @@ class AdministrationPanelStrategy extends AbstractNavigationPanelStrategy
     const ROLE_ACCESS_UPDATE_USER = 'ROLE_ACCESS_UPDATE_USER';
     const ROLE_ACCESS_DELETE_USER = 'ROLE_ACCESS_DELETE_USER';
     const ROLE_ACCESS_ROLE = 'ROLE_ACCESS_ROLE';
+    const ROLE_ACCESS_CREATE_ROLE = 'ROLE_ACCESS_CREATE_ROLE';
+    const ROLE_ACCESS_UPDATE_ROLE = 'ROLE_ACCESS_UPDATE_ROLE';
+    const ROLE_ACCESS_DELETE_ROLE = 'ROLE_ACCESS_DELETE_ROLE';
```
Comment (itkg-ngilain) [inline Backoffice/NavigationPanel/Strategies/AdministrationPanelStrategy.php]: @Thiblef : no access role to see roles ?

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**